examples/stickysession/nginx/README.md

```diff
@@ -0,0 +1,58 @@
+# Sticky Session
+
+This example demonstrates how to Stickness in a Ingress.
+
+## Prerequisites
+
+You will need to make sure you Ingress targets exactly one Ingress
+controller by specifying the [ingress.class annotation](/examples/PREREQUISITES.md#ingress-class),
+and that you have an ingress controller [running](/examples/deployment) in your cluster.
+
+Also, you need to have a deployment with replica > 1. Using a deployment with only one replica doesn't set the 'sticky' cookie.
+
+## Deployment
+
```

Comment:
Suggest something like: 

Session stickyness is achieved through 3 annotations on the Ingress, as shown in the [example](sticky-ingress.yaml). 

Name | Description | Values
---------|-----------------|------------
ingress.kubernetes.io/sticky-enabled annotation |  enables stickyness | true/false 
ingress.kubernetes.io/sticky-name |  (does what?) | route, (and what are the possible values)
ingress.kubernetes.io/sticky-hash | does what? | sha1 (does what, and what are the possible values)

You can create the ingress to test this
```
$ kubectl create -f 
```